### PR TITLE
[IMP] web_responsive: Better support for long titles

### PR DESCRIPTION
**Pull request by Tardo**
long_title

Rules used:

* Small screens uses 100%
* Other screen sizes uses 100% - (132px * 3 + Some margin) => Size of button * max colums (+ added some margin) [132px it's a hardcoded size]

Cc @tecnativa

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]
Although not sure if `!important` is wasting other things

File: web_responsive/static/src/css/web_responsive.scss
Comment (yajo) [inline web_responsive/static/src/css/web_responsive.scss]: I tested this and it seems to have the same effect, so as a good rule of thumb, always avoid `!important` clauses:

```suggestion
                width: calc(100% - 450px);
```

File: web_responsive/static/src/css/web_responsive.scss
Comment (yajo) [inline web_responsive/static/src/css/web_responsive.scss]: It seems like this `!important` should stay, however. Unless, maybe, if we switch to top full button row as suggested above.

**Review by yajo — CHANGES_REQUESTED** [review]
On wide screens we get this strange behavior:

Captura de pantalla de 2019-10-02 11-13-10

I found interesting how this is handled in enterprise:

Captura de pantalla de 2019-10-02 11-14-47

It seems to me that, if we make the button box 100% wide as a top row, just like enterprise, then we can set oe_title also to 100% width always (no matter screen size), and we can get more comfortable and consistent form screens. As an example see the difference here:

Peek 02-10-2019 11-20

Versus this (I added all those "Active" buttons just to trigger the dropdown):

Peek 02-10-2019 11-32


Not sure what you think, but I think the enterprise one is more pleasant.

Well, just take this as an idea. The current behavior is not "wrong", and I wouldn't block this PR if you prefer it like it is. On that subject, just see the comments below.

**Comment by LoisRForgeFlow** [comment]
For me @Yajo proposal is the best, also note that in v13 CE the button box looks like this with the top of the form dedicated to it.

**Comment by pedrobaeza** [comment]
Yeah, but that means deeper changes. What do you think, @Tardo?

**Comment by yajo** [comment]
@lreficent Thanks, I didn't notice this was also like this in vanilla v13 CE:

Captura de pantalla de 2019-10-09 13-21-39


~~Then AFAICS it would be actually *less* work to leave that part as it is upstream, right? I mean, it would mean actually code removal.~~ :point_left: Sorry, I didn't realize this PR is for v12. Then, maybe you can add that to the roadmap for v13, and accept this PR as it is for v12.

**Comment by Tardo** [comment]
Ok, changes done... i know that it's not the best solution but is one that not require a lot of changes  in css and js.

**Review by GSLabIt — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Wrong rebase with extra unwanted commit

**Comment by Tardo** [comment]
Oops! sorry, fixed commit history.

**Review by LoisRForgeFlow — APPROVED** [review]
Ok, this is already an improvement.

Thank you!

**Comment by pedrobaeza** [comment]
@Yajo your call

**Review by yajo — APPROVED** [review]

**Pull request merged**
